.vscode/launch.json

```diff
@@ -64,87 +61,87 @@
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Sync All",
             "args": ["--sync", "all", "--debugpath", "${input:debugPath}", "--debugbranch", "${input:debugBranch}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Push Binaries",
             "args": ["--push", "Binaries.zip", "--debugpath", "${input:debugPath}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Pull Binaries",
             "args": ["--sync", "binaries", "--debugpath", "${input:debugPath}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Pull Engine Build",
             "args": ["--sync", "engine", "--bundle", "${input:bundleSelection}", "--debugpath", "${input:debugPath}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Generate DDC",
             "args": ["--sync", "ddc", "--debugpath", "${input:debugPath}"],
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Update Engine Version to Latest",
```

Comment:
While running this configuration, program gives this exception:

```
Exception has occurred: FileNotFoundError
[WinError 2] The system cannot find the file specified
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbpy\pbtools.py", line 25, in run_with_combined_output
    return subprocess.run(*cmd, text=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbpy\pbtools.py", line 29, in get_combined_output
    return run_with_combined_output(*cmd).stderr
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbpy\pbunreal.py", line 175, in get_latest_available_engine_version
    output = pbtools.get_combined_output(["gsutil", "ls", bucket_url])
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbsync\pbsync.py", line 167, in sync_handler
    engine_version = pbunreal.get_latest_available_engine_version(
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbsync\pbsync.py", line 340, in main
    sync_handler(args.sync, args.repository, args.bundle)
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbsync\pbsync.py", line 359, in <module>
    main()
```

Reply:
Can you hold off on testing these? The subprocess isn't ready yet.

Reply:
Let me know when it's ready then, because I was told the PR can be approved after fetch steps are reverted.

Reply:
I then made additional changes after I said that. Sorry for the confusion.

Reply:
Should be fixed, test again.

Reply:
```
Exception has occurred: FileNotFoundError
[WinError 2] The system cannot find the file specified
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbpy\pbtools.py", line 25, in run_with_combined_output
    return subprocess.run(*cmd, text=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbpy\pbtools.py", line 29, in get_combined_output
    return run_with_combined_output(*cmd).stdout
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbpy\pbunreal.py", line 175, in get_latest_available_engine_version
    output = pbtools.get_combined_output(["gsutil", "ls", bucket_url])
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbsync\pbsync.py", line 167, in sync_handler
    engine_version = pbunreal.get_latest_available_engine_version(
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbsync\pbsync.py", line 338, in main
    sync_handler(args.sync, args.repository, args.bundle)
  File "C:\Users\dyanikoglu\Desktop\PBSync\pbsync\pbsync.py", line 357, in <module>
    main()
```

Are you able to run those configurations on vs code?

Reply:
Should be fixed now